src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1575,6 +1575,40 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
         [Theory]
         [InlineData(false)]
         [InlineData(true)]
+        [InlineData(null)]
+        public async Task PostAsync_ExpectContinue_Success(bool? expectContinue)
```

Comment:
It's supposed to not fail; the relevant asserts are guarded by platform/handler. If there are others we should add to ensure those asserts run on additional platforms, we can.